app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala

```diff
@@ -137,7 +137,7 @@ class ApplicationService @Inject()(applicationRepository: ApplicationRepository,
     fetchApp(applicationId) flatMap { app =>
 
       def updateWso2Subscriptions(): Future[Seq[HasSucceeded]] = {
-        apiGatewayStore.getSubscriptions(app.wso2Username, app.wso2Password, app.wso2ApplicationName) flatMap { originalApis =>
+        subscriptionRepository.getSubscriptions(applicationId) flatMap { originalApis =>
```

Comment:
I don't think we need this change, as this entire method (updateWso2Subscriptions) will be removed once we switch over to AWS (updating an app to another tier won't require to update the subscriptions).

Reply:
Have reverted that now